src/minuteman_ipvs_mgr.erl

```diff
@@ -280,23 +313,32 @@ code_change(_OldVsn, State, _Extra) ->
 %%%===================================================================
 %%% Internal functions
 %%%===================================================================
-handle_get_services(#state{netlink_generic = Pid, family = Family}) ->
-    AF = netlink_codec:family_to_int(inet),
-    Protocol = netlink_codec:protocol_to_int(tcp),
+handle_get_services(State) ->
+    lists:foldl(
+      fun(Protocol, Acc) ->
+              Acc ++ handle_get_services(inet, Protocol, State)
```

Comment:
Can we break this in two lines? For example Services = handle_get_services(inet, Protocol, State), [Services|Acc]. I am wondering why we used "++". Do we care about the order in the list?